recipes-connectivity/openssl/openssl_1.1.1g.bbappend

```diff
@@ -0,0 +1 @@
+SRC_URI_remove = "file://debian/pic.patch"
```

Comment:
Could you add a comment in this `.bbappend` which explains why we're removing this `.patch`. i think that would be easier than having our future selves dig back through the git log to rediscover the rationale.

Reply:
Done.